pages/_en-US/ds-index/ds-bios-firmware-dump.md

```diff
@@ -0,0 +1,75 @@
+---
+lang: en-US
+layout: wiki
+section: ds-index
+category: guides
+title: BIOS / Firmware Dumping
+description: How to extract the Nintendo DS or DSi BIOS / Firmware from your console
+tabs:
+  - tab-twl-sd-card: DSi / 3DS with custom firmware
+    tab-flashcard: Flashcard
+---
+
+Some emulators, such as MelonDS, require the DS-mode or DSi-mode BIOS and firmware, which is only available by extracting from your console. There are two distinct ways to achieve this, being split into a DSi or 3DS with custom firmware, or a DS / DS Lite / DSi or 3DS without custom firmware and a flashcard compatible with the console.
+
+The DSi / 3DS method will extract both the DS-mode and DSi-mode BIOS and firmware files. The Flashcard method will only extract the DS-mode files. It is highly recommended to use the DSi / 3DS method where possible.
```

Comment:
The 3DS cannot extract full DSi mode BIOS files, at best they *might* work for some things, but they will be the incomplete dumps missing RSA keys

Reply:
No longer using dsibiosdumper on 3DS